src/main/java/org/broadinstitute/hellbender/tools/spark/sv/BreakpointComplications.java

```diff
@@ -48,6 +42,16 @@
     private List<String> cigarStringsForDupSeqOnCtg = null;
     private boolean dupAnnotIsFromOptimization = false;
 
+    public String onErrStringRep() {
```

Comment:
It used to have the `toString()` overridden, this change was to signify that this methods is to be used for human consumption when things go wrong, hence the name "on error string representation"